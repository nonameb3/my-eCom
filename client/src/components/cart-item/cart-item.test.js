import React from "react";
import { shallow } from "enzyme";

import CartItem from "./cart-item.component";

const mockProps = {
  item: {
    imageUrl:
      "https://images.unsplash.com/photo-1495105787522-5334e3ffa0ef?ixlib=rb-1.2.1&auto=format&fit=crop&w=1950&q=80",
    price: 99,
    name: "jacket",
    quantity: 10
  }
};

it("should render cartItem compoent", () => {
  const wrapper = shallow(<CartItem {...mockProps} />);
  expect(wrapper).toMatchSnapshot();
});
