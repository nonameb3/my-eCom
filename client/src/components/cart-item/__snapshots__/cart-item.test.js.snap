// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render cartItem compoent 1`] = `ShallowWrapper {}`;
